src/app/client/src/app/modules/workspace/components/up-for-review/up-for-review.component.ts

```diff
@@ -223,15 +223,17 @@ export class UpForReviewComponent extends WorkSpace implements OnInit {
         subject: bothParams.queryParams.subject,
         medium: bothParams.queryParams.medium,
         gradeLevel: bothParams.queryParams.gradeLevel,
-        contentType: bothParams.queryParams.contentType ? bothParams.queryParams.contentType : this.config.appConfig.WORKSPACE.contentType
       },
       limit: limit,
       offset: (pageNumber - 1) * (limit),
       query: _.toString(bothParams.queryParams.query),
       sort_by: this.sort
     };
-    const contentType = this.getContentType && this.getContentType().contentType;
-    searchParams.filters.contentType = contentType;
+    if (!_.get(bothParams.queryParams, 'contentType')) {
+      searchParams.filters['contentType'] = this.getContentType && this.getContentType();
+    } else {
+      searchParams.filters['contentType'] =  bothParams.queryParams.contentType;
+    }
```

Comment:
searchParams.filters['contentType'] = _.get(bothParams.queryParams, 'contentType') ||  this.getContentType();